src/client/components/SignupForm/index.js

```diff
@@ -0,0 +1,22 @@
+const React = require('react');
+
+const SignupForm = (props) => (
+  <form required className="signup-form" method="POST" action="/api/signup"
+    onSubmit={props.onSubmit}>
+    <input type="text" className="textInput" name="username"
```

Comment:
why do you have textInput class? You can use in the css attributes too for example:
input[type="text"], 
input[type="email"],
input[type="password"],

or just simple:
input:not([type="submit"])